app/controllers/orders_controller.rb

```diff
@@ -1,10 +1,15 @@
 class OrdersController < ApplicationController
+  def index
+    @orders = Order.where(employee_id: current_employee)
+  end
+
   def new
     @order = Order.new
   end
 
   def create
     @order = Order.new(order_params)
+    @order.points = reward.price
```

Comment:
I think `points` might be a bit misleading - I would use something like `purchuase_price` or `value`
```suggestion
    @order.purchuase_price = reward.price
```